examples/gno.land/p/demo/fqname/fqname_test.gno

```diff
@@ -72,3 +72,22 @@ func TestRenderLink(t *testing.T) {
 		uassert.Equal(t, tt.expected, result, "Rendered link did not match expected")
 	}
 }
+
+func TestShortPath(t *testing.T) {
+	tests := []struct {
+		pkgPath  string
+		expected string
+	}{
+		{"gno.land/p/demo/avl", "/p/demo/avl"},
+		{"github.com/a/b", "/a/b"},
+		{"example.com/pkg", "/pkg"},
+		{"gno.land/r/demo/foo20", "/r/demo/foo20"},
+		{"gno.land/r/demo/foo20", "/r/demo/foo20"},
+		{"", ""},
```

Comment:
`gno.land/p/demo/avl?foo=test1&bar=test2`?

Reply:
`/p/demo/avl`?

Reply:
Yeah. This whole package needs to be tested better; I'll add a few cases and more docs about the func

Reply:
rm'd my addition to fqname; i just needed this simple happy-path flow, which is happening 100% of the time in my case. [c685a46](https://github.com/gnolang/gno/pull/2842/commits/c685a469359c6c29173a194f9f1e88d3b3d1ae08)